### ISSUE TITLE
App's language doesn't change

### ISSUE THREAD

**Issue by alcmene00** [UriahShaulMandel/BaldPhone#803]
**Describe the bug**
When I download the app and my system’s language is in English, if I want to change the app’s language to Greek, the app’s language doesn’t change, while my system’s language changes to Greek. I tried to change to French, Italian and it doesn’t change, too. 
However, if I download the app and my system’s language is Greek, the app’s language changes.

**To Reproduce**
1.	System language English (United States)
2.	Download the app from google play (version 13.0.0)
3.	Go on BaldPhone’s settings, and change the language to Greek/ French/ Italian (settings -> personalization -> language -> Ελληνικά)
4.	The app’s language is still in English.

**Expected behavior**
When the system's language changes, the app's language has to change too.

**Screenshots**
In those screenshots, you can see that my system’s language has change, but the app’s not.
![1](https://user-images.githubusercontent.com/45488168/104844250-d504b300-58d7-11eb-8961-1eb0290a9ad1.jpg)
![2](https://user-images.githubusercontent.com/45488168/104844251-d635e000-58d7-11eb-9267-eca0c64c4d98.jpg)
![3](https://user-images.githubusercontent.com/45488168/104844252-d6ce7680-58d7-11eb-8664-b25228c88894.jpg)

**Phone**
- BaldPhone Version: 13.0.0 
- Android Version: 9
- Device: Redmi 8

**Comment by UriahShaulMandel** [comment]
Hey there!
And thank you so much for the detailed issue!
it really helps to solve solutions that way.
I can't reproduce the bug on my device,
does it still not work for you? cant you try on another device?
That is really weird!

**Comment by alcmene00** [comment]
I told people I know to try to reproduce the bug. 
The devices that tried to reproduce it where:
Xiaomi Redmi note 8 with android version 10, Xiaomi Redmi note 9 Pro with android version 10, Xiaomi Redmi 5A with android version 8.1.0, Samsung galaxy a20e with android version 9 and Samsung galaxy A71 with android 10.  Everyone downloaded the 13.0.0 version from Google Play.
It seems like the bug is happening in Xiaomi Redmi and not in Samsung devices. The system language was changing in every device, but in Xiaomi the app’s language was still in English.
What device do you have?

**Comment by UriahShaulMandel** [comment]
Thank you so much for the amount of testing!
I've tried it on many android devices, but not xianomi. Do you have this issue with other apps too?
Thank you!

**Comment by alcmene00** [comment]
No, I don't think so. Every app has the same language as my system language (if it has translation to my system language).

**Comment by UriahShaulMandel** [comment]
Have you tried a restart of the phone? (After changing the language)

**Comment by alcmene00** [comment]
Yes, I have restared my phone and the bug remains
